.travis.yml

```diff
@@ -1,10 +1,17 @@
 language: python
+sudo: required
+dist: trusty
 python:
   - "3.5"
+before_install:
+  - sudo apt-get -qq update
+  - sudo apt-get install build-essential libssl-dev libffi-dev python-dev
+
 # command to install dependencies
 install:
   - pip install -r requirements.txt
   - pip install -r requirements_for_test.txt
+  - pip install cryptography==1.1.2 --upgrade --force
```

Comment:
Is this still needed? We've determined that it wasn't the version that was causing the problem and requirements.txt should contain the correct version now (1.1.2), confirm that and remove this?